src/components/navigation/GlobalNavigation/NotificationCenter.tsx

```diff
@@ -0,0 +1,58 @@
+import { Flex, Typography, Space, Button } from 'antd'
+import { Icon, type IPopoverProps, Popover } from 'src/components'
+import { NavigationItem } from 'src/components/navigation/GlobalNavigation/NavigationItem'
+
+export type INotificationOptions = Omit<
+  IPopoverProps,
+  'key' | 'trigger' | 'placement' | 'arrow' | 'overlayClassName' | 'title'
+>
+export interface INotificationActions {
+  onClose?: () => void
+  onPreferencesClick?: () => void
+}
```

Comment:
Since we are dealing with library code it's important to forward the original events that triggered those actions so users are able to access it if they need it. Since those are button clicks, we can reference the props from there to have matching types:

```suggestion
export interface INotificationActions {
  onClose?: IButtonProps["onClick"]
  onPreferencesClick?: IButtonProps["onClick"]
}
```

Reply:
fixed